metagraph/benchmarks/graph/bench_boss.cpp

```diff
@@ -14,17 +15,47 @@ using mtg::graph::boss::BOSS;
 const std::string filename = "./benchmark_graph.dbg";
 
 constexpr uint64_t NUM_DISTINCT_INDEXES = 1 << 21;
+constexpr uint64_t PATH_SIZE = 1 << 7;
+constexpr uint64_t CACHE_SIZE = 1 << 10;
```

Comment:
Just write the number, why do we need these powers if it's just 1024 or something smaller?